docs/config.c

```diff
@@ -2210,6 +2210,43 @@
 ** how often (in seconds) NeoMutt will update message counts.
 */
 
+{ "mailbox_folder_format", DT_STRING, "%2C %t %N %n %m %i" },
+/*
+** .pp
+** This variable allows you to customize the file browser display to your
+** personal taste. It's only used to customize network mailboxes (e.g. imap).
+** This string is similar to $$index_format, but has its own set of
+** \fCprintf(3)\fP-like sequences:
+** .dl
+** .dt %C  .dd   .dd Current file number
+** .dt %d  .dd   .dd Date/time folder was last modified
+** .dt %D  .dd   .dd Date/time folder was last modified using $$date_format.
+** .dt %f  .dd   .dd Filename ("/" is appended to directory names,
+**                   "@" to symbolic links and "*" to executable files)
+** .dt %F  .dd   .dd File permissions
+** .dt %g  .dd   .dd Group name (or numeric gid, if missing)
+** .dt %i  .dd   .dd Description of the folder
+** .dt %l  .dd   .dd Number of hard links
+** .dt %m  .dd * .dd Number of messages in the mailbox
+** .dt %n  .dd * .dd Number of unread messages in the mailbox
+** .dt %N  .dd   .dd "N" if mailbox has new mail, blank otherwise
+** .dt %s  .dd   .dd Size in bytes (see $formatstrings-size)
+** .dt %t  .dd   .dd "*" if the file is tagged, blank otherwise
```

Comment:
It looks to me like mailboxes can't be tagged.

Reply:
Should we remove the options shared with `folder_format_str()` that can't be used in mailboxes from the documentation?

Reply:
No, we can remove this line of doc.
NeoMutt will still accept `%t`, but it'll always be empty.

Reply:
Are there other fields that can be removed too (like gid, uid, etc)? Because then I can make the changes altogether